packages/web/src/pages/upload-page/fields/StemFilesView.tsx

```diff
@@ -73,13 +73,16 @@ export const StemFilesView = ({
               index={i}
               displayIndex={stems.length > 1}
               key={`stem-${i}`}
-              trackTitle={stem.file.name}
-              fileType={stem.file.type}
-              fileSize={stem.file.size}
+              trackTitle={stem.file?.name ?? stem.metadata.orig_filename ?? ''}
+              fileType={stem.file?.type ?? ''} // TODO: Get correct file type for pre-existing stems
```

Comment:
in this PR, for the edit flow, the `TrackPreviewNew` component guesses the file type of previously existing stems based on the file extension (if filename has it), otherwise it returns the unknown file type icon.
we can, as a fast-follow, make a change to use the file type from the '/inspect' dn endpoint, as we do for file size (i didnt realize this was already available when making the changes).

same comment for the size on the line below